reaper/util.py

```diff
@@ -120,7 +120,7 @@ def write_state_file(path, state):
     os.rename(temp_path, path)
 
 
-def create_archive(content, arcname, metadata=None, outdir=None):
+def create_archive(content, arcname, rootdir=True, metadata=None, outdir=None):
```

Comment:
If we end up needing this additional parameter, we should add it to the end of the list. As is, this breaks some of the other tools that rely on parameter positions rather than names.